anyway/language_constants.py

```diff
@@ -0,0 +1,34 @@
+class Hebrew(object):
+    ROAD = "כביש"
+    INTERCITY = "בינעירוני"
+
+    def __init__(self, *args, **kwargs):
+        super(Hebrew, self).__init__(*args, **kwargs)
+
+    @staticmethod
+    def concat(*words, sep=" "):
+        return sep.join(words)
+
+    @staticmethod
+    def IN(word):
+        return "ב" + word
```

Comment:
Why use non-standard ALL_CAPS?

Reply:
because `in` is a reserved word

Reply:
That is not, by itself, enough of a reason. You could use `in_` or `at`.